docs/references/remotes.md

```diff
@@ -0,0 +1,283 @@
+>
+
+Example:
+```python
+chain = DataChain.from_storage(
+    "s3://my-bucket/my-dir",
+    client_config = {
+		"endpoint_url": "<minio-endpoint-url>",
+		"key": "<minio-access-key",
+		"secret": "<minio-secret-key"
+	}
+)
+```
+
+### Google Cloud Storage
+
+DataChain uses [gcsfs](https://gcsfs.readthedocs.io/en/latest/) to interact with Google Cloud Storage. Authentication can be achieved by using any of the method described at [gcsfs documentation](https://gcsfs.readthedocs.io/en/latest/#credentials). You can also pass the following configuration parameters to the gcsfs filesystem as client_config dictionary.
+
+>	- project : string
+>        project_id to work under. Note that this is not the same as, but often
+>        very similar to, the project name.
+>        This is required in order
+>        to list all the buckets you have access to within a project and to
+>        create/delete buckets, or update their access policies.
+>        If ``token='google_default'``, the value is overridden by the default,
+>        if ``token='anon'``, the value is ignored.
+>
+>    - access : one of {'read_only', 'read_write', 'full_control'}
+>        Full control implies read/write as well as modifying metadata,
+>        e.g., access control.
+>
+>    - token: None, dict or string
+>        (see description of authentication methods, from link above)
+>
+>    - consistency: 'none', 'size', 'md5'
+>        Check method when writing files. Can be overridden in open().
+>
+>    - cache_timeout: float, seconds
+>        Cache expiration time in seconds for object metadata cache.
+>        Set cache_timeout <= 0 for no caching, None for no cache expiration.
+>
+>    - secure_serialize: bool (deprecated)
+>
+>    - requester_pays : bool, or str default False
+>        Whether to use requester-pays requests. This will include your
+>        project ID `project` in requests as the `userProject`, and you'll be
+>        billed for accessing data from requester-pays buckets. Optionally,
+>        pass a project-id here as a string to use that as the `userProject`.
+>
+>    - session_kwargs: dict
+>        passed on to ``aiohttp.ClientSession``; can contain, for example,
+>        proxy settings.
+>
+>    - endpoint_url: str
+>        If given, use this URL (format protocol://host:port , *without* any
+>        path part) for communication. If not given, defaults to the value
+>        of environment variable "STORAGE_EMULATOR_HOST"; if that is not set
+>        either, will use the standard Google endpoint.
+>
+>    - default_location: str
+>        Default location where buckets are created, like 'US' or 'EUROPE-WEST3'.
+>        You can find a list of all available locations here:
+>        https://cloud.google.com/storage/docs/locations#available-locations
+>
+>    - version_aware: bool
+>        Whether to support object versioning. If enabled this will require the
+>        user to have the necessary permissions for dealing with versioned objects.
+
```

Comment:
```suggestion
- `version_aware`: `bool` (default: `False`)

    Whether to support object versioning. If enabled this will require the
    user to have the necessary permissions for dealing with versioned objects.


```